ethers-solc/src/config.rs

```diff
@@ -550,6 +550,12 @@ impl ArtifactOutput for MinimalCombinedArtifactsHardhatFallback {
 #[derive(Clone, Debug, Default)]
 pub struct AllowedLibPaths(pub(crate) Vec<PathBuf>);
 
+impl AllowedLibPaths {
+    pub fn is_empty(&self) -> bool {
+        self.0.is_empty()
+    }
```

Comment:
iirc there's a clippy lint that checks that `fn is_empty` and `fn len` come in pairs.
alternatively, we can `impl AsRef` and/or `AsMut`, or simply make it pub